src/workspace-data/data-table/entity-service/SavedColumnSettings.js

```diff
@@ -200,8 +200,8 @@ const SavedColumnSettings = ({ workspace, snapshotName, entityType, entityMetada
   });
 
   const selectedSettingsNameExists = _.has(selectedSettingsName, savedColumnSettings);
-  const editWorkspaceErrorMessage = workspace ? editWorkspaceError(workspace) : undefined;
-  const canSaveSettings = workspace && !editWorkspaceErrorMessage;
+  const { value: canEdit, message: editErrorMessage } = canEditWorkspace(workspace);
```

Comment:
This drops the check if workspace is truthy. Is that not needed?

Reply:
https://github.com/DataBiosphere/terra-ui/pull/4268/commits/060b082b57f2c41efd47d204bd4f81d63caf0f3f